fluent-tests/report.md

```diff
@@ -1,5 +1,5 @@
 # Java Codegen Report
```

Comment:
this file does not matter, you do not need to merge or else, you can just delete it if conflict.